Source/WebKit/Scripts/PreferencesTemplates/WebPreferencesFeatures.cpp.erb

```diff
@@ -37,6 +37,7 @@ namespace WebKit {
 
 const Vector<RefPtr<API::Object>>& WebPreferences::features()
 {
+    WTF::initialize(); // We use WTF strings.
```

Comment:
This looks like it may be in the wrong place.  I would expect InitializeWebKit2() to take care of this.

Reply:
This gets called from:
```7   WebKit                        	       0x1083ac780 WebKit::WebPreferences::features() + 152 (WebPreferencesFeatures.cpp:42)
8   WebKit                        	       0x107ff6bf0 WebKit::WebPreferences::experimentalFeatures()::$_74::operator()() const + 36 (WebPreferences.cpp:85)
9   WebKit                        	       0x1084dc5a8 WebKit::WebPreferences::experimentalFeatures() + 64 (WebPreferences.cpp:83)
10  WebKit                        	       0x1084801dc +[WKPreferences(WKPrivate) _experimentalFeatures] + 24 (WKPreferences.mm:564)
11  MiniBrowser                   	       0x104a53f14 -[SettingsController _populateMenu:] + 1772 (SettingsController.m:207)
12  MiniBrowser                   	       0x104a537b4 -[SettingsController initWithMenu:] + 436 (SettingsController.m:109)
13  MiniBrowser                   	       0x104a50844 -[BrowserAppDelegate awakeFromNib] + 40 (AppDelegate.m:83)
14  CoreFoundation                	       0x18701a5f0 -[NSSet makeObjectsPerformSelector:] + 176
15  AppKit                        	       0x18a22631c -[NSIBObjectData nibInstantiateWithOwner:options:topLevelObjects:] + 824
16  AppKit                        	       0x18a21be9c loadNib + 340
17  AppKit                        	       0x18a21b4cc +[NSBundle(NSNibLoading) _loadNibFile:nameTable:options:withZone:ownerBundle:] + 560
18  AppKit                        	       0x18a21b1d0 -[NSBundle(NSNibLoading) loadNibNamed:owner:topLevelObjects:] + 180
19  AppKit                        	       0x18a21aff4 +[NSBundle(NSNibLoading) loadNibNamed:owner:] + 300
20  AppKit                        	       0x18a20e750 NSApplicationMain + 488
21  MiniBrowser                   	       0x104a535a8 main + 48 (main.m:33)
22  dyld                          	       0x186beff28 start + 2236```

Reply:
Offline Alex suggested that we should be calling InitializeWebkit2 from anywhere that calls into _experimentalFeatures and its kin instead

Reply:
done